.storybook/webpack.config.js

```diff
@@ -2,19 +2,35 @@ const path = require('path');
 
 module.exports = {
   module: {
-    loaders: [
+    rules: [
       {
-        test:   /\.scss$/,
-        loaders: ['style', 'raw', 'sass'],
-        include: path.resolve(__dirname, '../css/')
+        test: /\.scss$/,
+        use: ['style-loader', 'raw-loader', 'sass-loader'],
+        include: [
+          path.resolve(__dirname, './'),
```

Comment:
Yes, so this is looking in the `/.storybook` config folder, which contains the scss for the storybook whereas the next line looks for the `react-dates` scss files, which are located in the `/css` folder